### DISCUSSION TITLE
React template using Tailwind, React Router and JSX+TW in content Script

### DISCUSSION THREAD

**Discussion by Ambushfall** [guocaoyi/create-chrome-ext#22] [Polls]
Simple idea, would anyone really like this?

(I know I do, as it helps me tremendously to not have to create elements one at a time vanilla way, and to style them super easily using Tailwind)

**Comment by guocaoyi** [comment]
In the beginning, it was only offered a pure template, includes Lang & Vite & UI Libs. For advance options, the cartesian product will be so large(Lang counts * css architecture * router * ui libs).

I'm starting up on JiNet(https://jizi.net, an agricultural information platform ) now, maybe no more time focus on this project.

so, by your mind! add router & css module or css arch for a new boilerplate. e.g. `template-react-ts-advence`
```bash
λ ~/ crx
✔ Project name: … my-crx-app
✔ Author: … ambushfall
✔ Framework: › react
✔ Language: ts
? Mode: › - Use arrow-keys. Return to submit.
❯   pure: lang & ui libs & vite
    advance: lang & ui libs & vite & router & css arch...
```
